browser-extension/mv2/src/background/background.js

```diff
@@ -571,6 +571,30 @@ BG.Methods.modifyRequestHeadersListener = function (details) {
 BG.Methods.onHeadersReceived = function (details) {
   var modifiedHeaders = BG.Methods.modifyHeaders(details.responseHeaders, RQ.HEADERS_TARGET.RESPONSE, details);
 
+  const initiatorPatterns = [
+    /^http:\/\/localhost:\d+/,
+    /^http:\/\/127\.0\.0\.1:\d+/,
+    /^https:\/\/app\.requestly\.io/,
+    /^https:\/\/beta\.requestly\.io/,
+  ];
+
+  const isSessionViewerInitiator = initiatorPatterns.some((initiator) => initiator.test(details.initiator));
```

Comment:
Can we name it `isAppInitiator` because by this method we are not able to exactly identify if sessionViewer is the initiator ?